codes/12865_py3_sjskgks.py

```diff
@@ -0,0 +1,13 @@
+import sys
+input = sys.stdin.readline
+
+numOfthings, limit = map(int, input().split())

```

Comment:
보통 대문자를 섞어서 네이밍 할때는 제일 앞 글자를 제외하고 의미마다 대문자를 씁니다. 그런데 꼭 반영할 필요는 없습니다.(반드시 지켜야할 규칙은 아니여서)
```suggestion
numOfThings, limit = map(int, input().split())

```